jsons/manifesto/it.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title":"Manifesto per l'Artigianato del Software",
+  "raising_the_bar":"Alzare l'asticella.",
+  "introduction":"Come aspiranti Artigiani del Software, stiamo alzando l'asticella dello sviluppo professionale del software praticandolo e aiutando gli altri a imparare il mestiere.  Attraverso questo lavoro siamo arrivati a dare valore a:",
+  "not_only":"Non solo ",
+  "but_also":"ma anche ",
+  "working_software":"software funzionante,",
+  "well_crafted_software":"software ben fatto",
+  "responding_to_change":"rispondere al cambiamento,",
+  "steadily_adding_value":"valore aggiunto costante",
+  "individuals_and_interactions":"individui e interazioni,",
+  "a_community_of_professionals":"una comunità di professionisti",
+  "customer_collaboration":"collaborazione con i clienti,",
+  "productive_partnerships":"partenariati produttivi",
+  "conclusion":"Cioè, nel perseguire gli elementi di sinistra abbiamo trovato indispensabili quelli di destra.",
```

Comment:
We can use "Questo é quanto" instead of "cioé", is it right as translation but looks strange as first word of a sentence in italian.

Reply:
Indeed "Cioè" is not good in that position but since the Manifesto is like someone it's reading it continually from the start until the end maybe "Ovvero" could be the right word because it's like to say "in altre parole".

What do you think?

Reply:
I like "Ovvero", I guess is the best compromise between adhering to the english version and choosing something that can be found at beginning of a sentence in Italian